openslides/urls.py

```diff
@@ -1,23 +1,27 @@
-from django.conf import settings
 from django.conf.urls import include, patterns, url
 
-from openslides.core.views import ErrorView
-from openslides.utils.plugins import get_urlpatterns
+from openslides.core.views import AngularView, ErrorView
 from openslides.utils.rest_api import router
 
+
+urlpatterns = patterns(
+    '',
+    url(r'^rest/', include(router.urls)),
+    # TODO: add "special" urls, for example pdf views etc.
+
+    url(r'^user.*', AngularView.as_view()),
```

Comment:
No, I want to keep "participants" in the GUI! The URL name is not so important for me. Currently, we use as url also "assignment" instead of "election" (like in GUI).